client/base_client.go

```diff
@@ -15,6 +15,7 @@ package pd
 
 import (
 	"context"
+	errs "github.com/pingcap/pd/v4/pkg/errors"
```

Comment:
em, why not name the package `github.com/pingcap/pd/v4/pkg/errs` directly?

Reply:
Originally, I used "errors" in #2658 . Then when I changed log code, I found github.com/pkg/errors was also used. So... = =

Reply:
Need to remove `pkg/errors`?

Reply:
`a`

Reply:
Some places use `pkg/errors` like `error.WithStack`.